src/sources/splunk_hec/mod.rs

```diff
@@ -2151,4 +2271,84 @@ mod tests {
             send_with(address, "services/collector/ack", message, TOKEN, &opts).await
         );
     }
+
+    #[test]
+    fn output_schema_definition_vector_namespace() {
+        let config = SplunkConfig {
+            log_namespace: Some(true),
+            ..Default::default()
+        };
+
+        let definition = config.outputs(LogNamespace::Vector)[0]
+            .clone()
+            .log_schema_definition
+            .unwrap();
+
+        let expected_definition = Definition::new_with_default_metadata(
+            Kind::object(Collection::empty()),
+            [LogNamespace::Vector],
+        )
+        .with_event_field(
+            &owned_value_path!("line"),
+            Kind::array(Collection::empty())
+                .or_object(Collection::empty())
+                .or_undefined(),
+            None,
+        )
+        .with_event_field(
+            &owned_value_path!("message"),
+            Kind::bytes().or_undefined(),
+            Some("message"),
+        )
```

Comment:
Ok, final question poking around this. In the Vector namespace would we want the event to either be "root is bytes" for `message` or "root is an object" for `lines`? If that's something we want to switch, now would be the time to do it.